lib/datadog/appsec/processor/context.rb

```diff
@@ -24,24 +26,22 @@ def run(input, timeout = WAF::LibDDWAF::DDWAF_RUN_TIMEOUT)
           start_ns = Core::Utils::Time.get_time(:nanosecond)
 
           input.reject! do |_, v|
-            case v
-            when TrueClass, FalseClass
-              false
-            else
-              v.nil? ? true : v.empty?
-            end
+            next false if v.is_a?(TrueClass) || v.is_a?(FalseClass)
```

Comment:
I think we actually don't need this line, and this whole `reject!` could be changed to:
```ruby
input.compact!.reject(&:empty?)
```

Reply:
and it's also fine to leave this block like it is, to reduce amount of changes in this PR